src/test/java/com/washingtonpost/arc/ans/v0_2/model/TestStory.java

```diff
@@ -8,7 +8,7 @@
 import org.junit.Test;
 
 /**
- * <p>Tests that JSON we expect to be a valid "Story" data file serializes correctly and validates
+ * <p>Tests that JSON we expect to be a valid "Collection" data file serializes correctly and validates
```

Comment:
roll this find & replace back... you shouldn't change anything under the ans/v0_2 directory because it's already released